packages/expo-build-properties/src/pluginConfig.ts

```diff
@@ -39,6 +39,7 @@ export interface PluginConfigType {
  */
 export interface PluginConfigTypeAndroid {
   /**
+   * @deprecated Set `newArchEnabled` through your app config file instead.
```

Comment:
```suggestion
   * @deprecated Use app config [`newArchEnabled`](https://docs.expo.dev/versions/unversioned/config/app/#newarchenabled) instead.
```

not sure whether the link is correct.
it would be good to point out the config name given expo-build-properties's `newArchEnabled` also specified inside app config file.

Reply:
Sure, updated

Reply:
still the old message?

Reply:
forgot to push the commit 🤦